File: problem_nineteen.py
# word count by input somplex sentence;
import re
text = input("Enter text:")
def MyFunction(text):
    value = re.sub(r'[,.!]', ' ', text)
    val =" ".join(value.split())
    print("Count = ",len(val.split(" ")))

MyFunction(text)

File: problem_nineteen.js
//word count by input somplex sentence;
var str = "hello , payer bhai ! . what are you doing "
function myFunction(val){
    var res = str.replace(/[.!, ]+/g, " ").trim().split(" ");
    console.log("count = ",res.length)
}
myFunction(str)

